labs/RivePlayer/src/CommunityToolkit.Labs.WinUI.Rive.RivePlayer.csproj

```diff
@@ -0,0 +1,41 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras/3.0.23">
+  <!-- Labs Constants -->
+  <Import Project="$(RepositoryDirectory)common\Labs.TargetFrameworks.props" />
+  <Import Project="$(RepositoryDirectory)common\Labs.ProjectIdentifiers.props" />
+
+  <!-- Labs Platform Config -->
+  <Import Project="$(RepositoryDirectory)common\Labs.Uno.props" />
+
+  <!-- Labs Project Config -->
+  <Import Project="$(RepositoryDirectory)common\Labs.MultiTarget.props" />
+
+  <PropertyGroup>
+    <RootNamespace>CommunityToolkit.Labs.WinUI.Rive</RootNamespace>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <PackageId>CommunityToolkit.Labs.$(PackageIdVariant).RivePlayer</PackageId>
+    <Description>
+      This package contains RivePlayer.
+    </Description>
+    <Version>0.0.1</Version>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Rive.RiveSharp" Version="1.0.1-alpha.c7588722d" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(IsUwp)' == 'true'">
+    <PackageReference Include="SkiaSharp.Views" Version="2.88.1" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(IsWinAppSdk)' == 'true'">
+    <PackageReference Include="SkiaSharp.Views.WinUI" Version="2.88.1" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(IsUno)' == 'true'">
+    <PackageReference Include="SkiaSharp.Views.Uno" Version="2.88.2" />
+  </ItemGroup>
```

Comment:
These should be in the `Dependencies.props` file vs. here. There's 4 specific packages that should be needed. (There'll be one for Uno + WindowsAppSDK that's missing here for instance.)

Reply:
Done!